### PR TITLE
[ng] toh-0: clarify that the live example is quickstart

### PR DESCRIPTION
**Pull request by chalin**
The new prose renders like this:
> screen shot 2017-06-08 at 8 53 13 am

### CODE REVIEW

File: src/angular/tutorial/toh-pt0.md [hunk 1]
```diff
@@ -11,15 +11,9 @@ nextpage:
 ---
 <?code-excerpt path-base="toh-0"?>
 This tutorial starts with a bare-bones Angular app.
-Run the <live-example name="quickstart">live example</live-example>
-to see the app.
-
-## Create the app
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: Why delete this heading? I'm afraid people will miss these instructions if they're buried in the first paragraph.

Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: I'm ok keeping the heading. Would you be ok with:
> This tutorial starts with a bare-bones Angular app.
>
> ## Create the app
>
> Create a project named `angular_tour_of_heroes`, using WebStorm or the command line and the \<live-example name="quickstart">angular-examples/quickstart\</live-example> GitHub project. For detailed instructions, see [Create a starter project](/angular/guide/setup#create-a-starter-project) from the [Setup for Development](/angular/guide/setup) page.

Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: Sure, if it renders correctly. :)

Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: Done.

**Comment by kwalrath** [comment]
I was going to say LGTM, but then I got to thinking...

The links before "GitHub project" are a little confusing. Is the first link to the running app or to the repo? Why is there a "view source" link here? Also, isn't it a "repo", not a "project"?

Also, I'd like the "For detailed instructions" to be in a separate paragraph, so the instructions are separated from the details that the reader might not need.

**Comment by chalin** [comment]
> Is the first link to the running app or to the repo?

The first link is to the running app.

> Why is there a "view source" link here?

That is the link to the repo.

> Also, isn't it a "repo", not a "project"?

Yes, I'll change that.

> Also, I'd like the "For detailed instructions" to be in a separate paragraph...

Ok

**Comment by kwalrath** [comment]
Here's a new screenshot to use: 
image

**Comment by chalin** [comment]
Closing since this is superseded by #720.

**Pull request closed without merging**